src/__tests__/Card.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import renderer from 'react-test-renderer'
+import 'jest-styled-components'
+import { Card, theme } from '..'
+
+describe('Card', () => {
+  test('renders small box shadow', () => {
+    const json = renderer.create(<Card boxShadowSize='sm' theme={theme} />).toJSON()
+    expect(json).toMatchSnapshot()
+    expect(json).toHaveStyleRule('box-shadow', '0 0 4px 0 rgba(0,0,0,0.08)')
+  })
+
+  test('renders medium box shadow', () => {
+    const json = renderer.create(<Card boxShadowSize='md' theme={theme} />).toJSON()
+    expect(json).toMatchSnapshot()
+    expect(json).toHaveStyleRule('box-shadow', '0 0 4px 0 rgba(0,0,0,0.08),0 8px 8px 0 rgba(0,0,0,0.08)')
```

Comment:
Jest doesn't seem to add the space from the separator in the `.join(', ')` when multiple shadows are joined together. Anyone know what might be the cause?
<img width="769" alt="screen shot 2017-10-11 at 12 43 30 pm" src="https://user-images.githubusercontent.com/6467349/31461243-f1cfab78-ae96-11e7-9110-e7ac76d5c814.png">

I modified the expected styles to not include the space to satisfy the tests, though would like to fix it if anyone have any insights. Thanks!

Reply:
Not for sure, But I guess the `toJSON()` method remove that space.

Reply:
My guess is this might be related to how [stylis](https://github.com/thysultan/stylis.js) parses the CSS, but not sure